incubator/exporter-stdout/src/main/java/io/aklivity/zilla/runtime/exporter/stdout/internal/stream/StdoutEventsStream.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright 2021-2023 Aklivity Inc
+ *
+ * Licensed under the Aklivity Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ *   https://www.aklivity.io/aklivity-community-license/
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package io.aklivity.zilla.runtime.exporter.stdout.internal.stream;
+
+import java.io.PrintStream;
+import java.util.function.LongFunction;
+import java.util.function.ToIntFunction;
+
+import org.agrona.DirectBuffer;
+import org.agrona.collections.Int2ObjectHashMap;
+
+import io.aklivity.zilla.runtime.engine.binding.function.MessageConsumer;
+import io.aklivity.zilla.runtime.engine.util.function.EventReader;
+
+public class StdoutEventsStream
+{
+    private final EventReader readEvent;
+    private final ToIntFunction<String> lookupLabelId;
```

Comment:
```suggestion
    private final ToIntFunction<String> supplyTypeId;
```

Reply:
renamed to `lookupTypeId`, pls see the comment above

Reply:
using supplyTypeId now as discussed